build/ls.py

```diff
@@ -26,6 +27,12 @@ def main(target_directory):
                       type=str,
                       required=True,
                       help="The target directory")
+  parser.add_argument("--file-type",
```

Comment:
Is this the file extension? If yes, could this flag be named `--file-extension`?

Reply:
ok